libteec/include/tee_client_api.h

```diff
@@ -293,16 +295,18 @@ typedef struct {
 	size_t size;
 	uint32_t flags;
 	/*
-	 * Implementation-Defined
+	 * Implementation defined
 	 */
-	int id;
-	size_t alloced_size;
-	void *shadow_buffer;
-	int registered_fd;
-	union {
-		bool dummy;
-		uint8_t flags;
-	} internal;
+	struct {
+		int id;
+		size_t alloced_size;
+		void *shadow_buffer;
+		int registered_fd;
+		union {
+			bool dummy;
+			uint8_t flags;
+		} internal;
```

Comment:
Is the `dummy` field still needed following this change? `dummy` was introduced in 4f3d4cbb7824bab8f5c0a445f301528a53142f6b not to break the ABI. Here we are breaking it anyway, so while at it, I suggest some clean up:
```suggestion
		uint8_t flags;
```
*N.B.:* If there is a wish to keep more flag bits for future use, then just declare `flags` as `uint32_t`.

Reply:
I agree. `uint32_t flags;` LGTM.

Reply:
@JLLinaro comment not addressed